src/open_inwoner/kvk/validators.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+# There are only 4 possibilities, let's enumerate them
+regexes = (
+    re.compile(r"^https://api.kvk.nl/api$"),
+    re.compile(r"^https://api.kvk.nl/api/$"),
+    re.compile(r"^https://api.kvk.nl/test/api$"),
+    re.compile(r"^https://api.kvk.nl/test/api/$"),
+)
```

Comment:
If we're going to restrict the API root to these values, I think it's better to define `choices` for the field and let people select one of these four.

I'm trying to think of a case where we'd want to use something else but can't really come up with one, is it necessary though to restrict the API root to these four values? It would be a hassle if we have to do a new release just to allow a different value here :thinking:

Reply:
I made the validation more general. I couldn't think of any reason for supporting other roots, but better safe than sorry.